src/globals/mixins.scss

```diff
@@ -55,3 +66,18 @@
       0 90%      /* bottom left */
     );
 }
+
+
+@mixin corner-clip-bottom-right() {
```

Comment:
@tomwpaterson-shape this isn't used anywhere. You sure we need it? If not, would mind deleting it from the code?